python/servo/platform/base.py

```diff
@@ -12,6 +12,9 @@
 import subprocess
 from typing import Optional
 
+# Nightly version that corresponds to stable specified in rust-toolchain.toml
+NIGHTLY_RUST = "nightly-2023-09-29"
```

Comment:
Can't we just (ab)use `RUSTC_BOOTSTRAP` instead of installing another toolchain?

Reply:
I've been using RUSTC_BOOTSTRAP with unstable rust flags like -Zbuild-std

Reply:
You are right. It works 🎉 !

Thanks for the notice.